__tests__/topic.tests.js

```diff
@@ -217,4 +217,98 @@ describe("POST /api/articles/:article_id/comments", () => {
         expect(response.body).toHaveProperty("postedComment");
       });
   });
+  describe("PATCH /api/articles/:article_id", () => {
+    test("Responds with votes(+) updated in the patched article", () => {
+      return request(app)
+        .patch("/api/articles/1")
+        .send({
+          inc_votes: 5,
+        })
+        .expect(201)
+        .then((response) => {
+          expect(response.body.article[0]).toMatchObject({
```

Comment:
You should just get a single article object back, not an array. Return rows[0] in your model to amend this